### PR TITLE
feat: Added functionality to the sidebar and small overlay improvements

### PR DESCRIPTION
**Pull request by AdrianSchulze**
- logout button
- sidebar functionality, map and filter owners channels
- empty test table and buttons
- link home, profile and logout
- show username through ME besides avatar

### CODE REVIEW

**Review by bitbytejoy — CHANGES_REQUESTED** [review]
Add a title to the PR.

When I start the app, I get the following errors in the console:

```
react-jsx-dev-runtime.development.js:87 Warning: Each child in a list should have a unique "key" prop.

Check the render method of `ProfilePicture`. See https://reactjs.org/link/warning-keys for more information.
    at LinkWithRef (http://localhost:3000/static/js/bundle.js:101256:7)
    at ProfilePicture (http://localhost:3000/static/js/bundle.js:1183:5)
    at div
    at http://localhost:3000/static/js/bundle.js:2929:66
    at Toolbar (http://localhost:3000/static/js/bundle.js:27999:82)
    at header
    at http://localhost:3000/static/js/bundle.js:2929:66
    at Paper (http://localhost:3000/static/js/bundle.js:23424:83)
    at http://localhost:3000/static/js/bundle.js:2929:66
    at AppBar (http://localhost:3000/static/js/bundle.js:8040:83)
    at Navbar (http://localhost:3000/static/js/bundle.js:1060:5)
    at div
    at http://localhost:3000/static/js/bundle.js:2929:66
    at Box (http://localhost:3000/static/js/bundle.js:32891:72)
    at Root (http://localhost:3000/static/js/bundle.js:198:68)
    at Auth (http://localhost:3000/static/js/bundle.js:393:5)
    at RenderedRoute (http://localhost:3000/static/js/bundle.js:102506:5)
    at Routes (http://localhost:3000/static/js/bundle.js:102972:5)
    at App
    at Router (http://localhost:3000/static/js/bundle.js:102910:15)
    at BrowserRouter (http://localhost:3000/static/js/bundle.js:101164:5)
printWarning @ react-jsx-dev-runtime.development.js:87
2ProfilePicture.tsx:36 Uncaught TypeError: Cannot read properties of null (reading 'username')
    at ProfilePicture (ProfilePicture.tsx:36:1)
    at renderWithHooks (react-dom.development.js:16305:1)
    at updateFunctionComponent (react-dom.development.js:19588:1)
    at beginWork (react-dom.development.js:21601:1)
    at HTMLUnknownElement.callCallback (react-dom.development.js:4164:1)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:4213:1)
    at invokeGuardedCallback (react-dom.development.js:4277:1)
    at beginWork$1 (react-dom.development.js:27451:1)
    at performUnitOfWork (react-dom.development.js:26557:1)
    at workLoopSync (react-dom.development.js:26466:1)
react-dom.development.js:18687 The above error occurred in the <ProfilePicture> component:

    at ProfilePicture (http://localhost:3000/static/js/bundle.js:1183:5)
    at div
    at http://localhost:3000/static/js/bundle.js:2929:66
    at Toolbar (http://localhost:3000/static/js/bundle.js:27999:82)
    at header
    at http://localhost:3000/static/js/bundle.js:2929:66
    at Paper (http://localhost:3000/static/js/bundle.js:23424:83)
    at http://localhost:3000/static/js/bundle.js:2929:66
    at AppBar (http://localhost:3000/static/js/bundle.js:8040:83)
    at Navbar (http://localhost:3000/static/js/bundle.js:1060:5)
    at div
    at http://localhost:3000/static/js/bundle.js:2929:66
    at Box (http://localhost:3000/static/js/bundle.js:32891:72)
    at Root (http://localhost:3000/static/js/bundle.js:198:68)
    at Auth (http://localhost:3000/static/js/bundle.js:393:5)
    at RenderedRoute (http://localhost:3000/static/js/bundle.js:102506:5)
    at Routes (http://localhost:3000/static/js/bundle.js:102972:5)
    at App
    at Router (http://localhost:3000/static/js/bundle.js:102910:15)
    at BrowserRouter (http://localhost:3000/static/js/bundle.js:101164:5)

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://reactjs.org/link/error-boundaries to learn more about error boundaries.
logCapturedError @ react-dom.development.js:18687
react-dom.development.js:26923 Uncaught TypeError: Cannot read properties of null (reading 'username')
    at ProfilePicture (ProfilePicture.tsx:36:1)
    at renderWithHooks (react-dom.development.js:16305:1)
    at updateFunctionComponent (react-dom.development.js:19588:1)
    at beginWork (react-dom.development.js:21601:1)
    at beginWork$1 (react-dom.development.js:27426:1)
    at performUnitOfWork (react-dom.development.js:26557:1)
    at workLoopSync (react-dom.development.js:26466:1)
    at renderRootSync (react-dom.development.js:26434:1)
    at recoverFromConcurrentError (react-dom.development.js:25850:1)
    at performConcurrentWorkOnRoot (react-dom.development.js:25750:1)
useChannel.tsx:30 Array(0)
useChannel.tsx:30 Array(0)
```

Errors will need to be fixed before approval.

**Comment by AdrianSchulze** [comment]
1. Changed the title.
2. Sonar is not implemented as I don't have any tests yet. I will implement it now.
3. Thanks for pointing out the errors. I wasn't aware of them.

**Review by bitbytejoy — APPROVED** [review]
lgtm

**Pull request merged**
